assets/src/edit-story/components/canvas/selectionCanvas.js

```diff
@@ -171,6 +172,21 @@ function SelectionCanvas({ children }) {
     }
   };
 
+  const createPolygon = useCallback((rotationAngle, x, y, width, height) => {
+    const {
+      topLeftPoint,
+      topRightPoint,
+      bottomRightPoint,
+      bottomLeftPoint,
+    } = getCorners(rotationAngle, x, y, width, height);
+    return new SAT.Polygon(new SAT.Vector(), [
+      new SAT.Vector(topLeftPoint.x, topLeftPoint.y),
+      new SAT.Vector(bottomLeftPoint.x, bottomLeftPoint.y),
+      new SAT.Vector(bottomRightPoint.x, bottomRightPoint.y),
+      new SAT.Vector(topRightPoint.x, topRightPoint.y),
+    ]);
+  }, []);
```

Comment:
This can just be moved out to a helper function - doesn't need to be inlined and redefined here every time this component initializes.